src/web3bch-wallet/Wallet.ts

```diff
@@ -335,4 +364,17 @@ export default class Wallet implements IWallet {
       })
       .catch((e) => { throw new ProviderException(e) })
   }
+
+  private isValidSignature(signature: string) {
+    const buf = Buffer.from(signature, "base64")
+    if (buf.length !== 65) {
+      return false
```

Comment:
I think hard coding the length is not possible here.

>This results in 71 bytes signatures (on average), as there are several header bytes, and the R and S valued are variable length.
https://bitcoin.stackexchange.com/questions/12554/why-the-signature-is-always-65-13232-bytes-long

Reply:
DER signatures
https://github.com/bitcoin/bips/blob/master/bip-0066.mediawiki